lib/services/theme.dart

```diff
@@ -0,0 +1,386 @@
+import "package:flutter/material.dart";
+
+class MaterialTheme {
```

Comment:
nit (non-blocking): mind adding a doc comment of how you generated this here? I guess on further thought that makes less sense, since we can't ensure that comment is still accurate if we adjust the colors afterwards.